core/extensions/VisioDemo/VisioDemo.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
I would remove the SLN file. They are more annoying than helpful most of the time. In this with only a single project they definitely aren't needed.

Reply:
That's fine.  Have added commit to remove the file.